test/spec/SearchApi/Support/GeoParserSpec.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace spec\SearchApi\Support;
+
+use SearchApi;
+use SearchApi\Models as Models;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+
+/**
+ * GeoParserSpec - Spec integration test for the Geo Parser Geocoder (higher level functions)
+ */
+class GeoParserSpec extends ObjectBehavior {
+  function it_is_initializable() {
+    $this->shouldHaveType( 'SearchApi\Support\GeoParser' );
+  }
+
+  function it_should_return_a_result() {
+  	$this->reverse_geocoder_parser( 'test' )->shouldBeString();
```

Comment:
I checked my code and it appears to have the correct indenting in code but git shows it over farther